script/release/release.js

```diff
@@ -139,6 +139,9 @@ function assetsForVersion (version, validatingRelease) {
     'electron-api.json',
     'electron.d.ts',
     'hunspell_dictionaries.zip',
+    'libc++.zip',
```

Comment:
For consistency we should probably ensure this zip is called `libcxx.zip`.  But we should probably call it something more descriptive.

`libcxx_objects.zip` is probably closer to the truth.

Also this artifact has to be per-platform so it will be similar to the other templated files in this list `libcxx-${version}-platform-arch.zip`